packages/enzyme-mocha/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "enzyme-mocha",
+  "version": "1.1.0",
+  "description": "Make Mocha and Enzyme play nice",
+  "main": "build/index.js",
+  "repository": "https://github.com/airbnb/enzyme/tree/master/packages/enzyme-mocha",
+  "keywords": [
+    "mocha",
+    "enzyme"
+  ],
+  "author": "Leland Richardson <leland.richardson@airbnb.com>",
+  "license": "MIT",
+  "dependencies": {
+    "jsdomify": "^1.0.2"
+  },
+  "peerDependencies": {
+    "react": "0.13.x || 0.14.x"
```

Comment:
it does depend on a global `describe` however :-/ so maybe it does need mocha?

I think it makes sense for a package called "foo-bar" to require foo and bar, even if they're not actually required. The goal is to prevent user mistakes, and when would someone want to install enzyme-mocha without both enzyme and mocha?